Task_DEV-1/FindTheMaxLenghtOfUniqueCharacters.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace Task_DEV_1
+{
+    /// <summary>
+    /// class that calculates maximum the number of unique not repeating consecutive characters
+    /// </summary>
+    class FindTheMaxLenghtOfUniqueCharacters
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                StringBuilder stringBuilderForSourceString = new StringBuilder();

```

Comment:
use only args[0], first arguments, no necessity to unite all arguments to one string